src/CoreWCF.Queue/src/CoreWCF/Queue/Common/QueueMessageContext.cs

```diff
@@ -14,6 +14,7 @@ namespace CoreWCF.Queue.Common
     public class QueueMessageContext : RequestContext
     {
         public PipeReader QueueMessageReader { get; set; }
+        public ReceiveContext ReceiveContext { get; set; }
         public virtual IDictionary<string, object> Properties { get { return _properties.Value; } }
```

Comment:
This shouldn't be public otherwise there's conflicting information. What if the request message doesn't have the receive context but this property does? I'd be okay with this being a helper property to add or get the value from the request message, but it shouldn't be an independent property backed by it's own value.